CHANGELOG.md

```diff
@@ -1,35 +1,42 @@
 # Change Log
 
+## [14.2.1] - 2025-04-13
+
+### Fixed
+
+- regression in 14.2.0 causing schema validation failure for generic-worker
+  tasks using run-task (#688)
+
 ## [14.2.0] - 2025-04-10
```

Comment:
Can you fix that date (and the one for 14.2.1). This was 2025-05-12. And it's not April anymore so 2025-05-13 for 14.2.1